README.md

```diff
@@ -1,7 +1,44 @@
-# Panther CLI Tool
-`panther-cli` is a Python command line interface for testing and packaging Panther Policies and Rules. Policies define the compliant and secure state of a cloud Resource, whereas Rules perform analysis on log data.
+# Panther Analysis
 
-## Installation
+This repository is a combination of open source policies and rules meant to be run with [Panther](https://github.com/panther-labs/panther), as well as the `panther-cli` tool which can be used to test and package these policies and rules without having to go through the Panther web interface.
+
+During initial deployments of Panther, the default configuration is to load all the policies and rules published during version releases of this project. This provides a set of out-of-the-box policies and rules to get analysis up and running. See the Panther documentation for how to override this default behavior if desired.
```

Comment:
During the initial deployment of Panther, all of the policies and rules published here are uploaded. This provides a set of out-of-the-box policies and rules to establish a strong detection baseline. See the [Panther documentation](https://docs.runpanther.io) for how to override this default behavior if desired.